recipes/llama-index-readers-file/meta.yaml

```
@@ -0,0 +1,52 @@
+{% set name = "llama-index-readers-file" %}
+{% set version = "0.1.13" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/llama_index_readers_file-{{ version }}.tar.gz
+    sha256: 830f06ec7b34437fc3bb5f268d235c5c7640296adb148d8f92277dceb7f0846d
+  # https://github.com/run-llama/llama_index/issues/10806
+  - url: https://raw.githubusercontent.com/run-llama/llama_index/main/LICENSE
+    sha256: 24f40b5190fdacabc24ddbb5f76364d15e4f030925220ea300d8a2dd4993c8cb
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python >=3.8,<4.0
+    - poetry-core
+    - pip
+  run:
+    - python >=3.8.1,<4.0
+    - llama-index-core >=0.10.1,<0.11.0
+    - beautifulsoup4 >=4.12.3,<5.0.0
+    - pypdf >=4.0.1,<5.0.0
+    - striprtf >=0.0.26,<0.0.27
+    - pymupdf <2.0.0,>=1.23.21
```

Theoretically this should only be a run export but the poetry metadata doesn't work correctly...
https://github.com/run-llama/llama_index/issues/12697

```suggestion
    # remove once https://github.com/run-llama/llama_index/issues/12697 is resolved
    - pymupdf <2.0.0,>=1.23.21
```

---

i'll add this comment in the generated feedstock afterwards @xhochy 😄

---

https://github.com/conda-forge/llama-index-readers-file-feedstock/commit/d4013ecaccc863eaf8a1ae7468c45290fc38ee35